plugin/src/main/java/com/playmonumenta/epicstructures/managers/RespawningStructure.java

```diff
@@ -278,8 +278,8 @@ public void run() {
 	}
 
 	public void forcedRespawn(Player player) {
-		if (player.hasMetadata("ForceReset")) {
-			player.removeMetadata("ForceReset", mPlugin);
+		if (player.hasMetadata("ForceResetPOI") && mOuterBounds.within(player.getLocation().toVector())) {
+			player.removeMetadata("ForceResetPOI", mPlugin);
 		} else {
 			player.sendMessage(ChatColor.RED + "" + ChatColor.BOLD + "Waited too long! Try again.");
 			return;
```

Comment:
I think probably want to change the error message here to "You must be nearby the POI to force its respawn". The way this is set up, the player will always have the metadata, because it's checked in the command itself to get the label, so it'll never be too slow if it got this far.